runner/runner.go

```diff
@@ -464,6 +473,78 @@ func runEvents(workerID int, events <-chan event) {
 	gEventWorkersRunning[workerID-1] = false
 }
 
+func runWpCmd(subcommand []string) (string, error) {
+	if fpm {
+		return runWpFpmCmd(subcommand)
+	} else {
+		return runWpCliCmd(subcommand)
+	}
+}
+
+func runWpFpmCmd(subcommand []string) (string, error) {
+	
+	path := fpmUrl.Path
+	host := fpmUrl.Host
+	
+	if path == "" || fpmUrl.Scheme == "unix" {
+		path = fpmUrl.Fragment
+	}
+	
+	if fpmUrl.Scheme == "unix" {
+		host = fpmUrl.Path
+	}
+	
+	fcgi, err := fcgiclient.Dial(fpmUrl.Scheme, host)
+	if err != nil {
+		log.Printf("fastcgi Dial failed: %s\n", err)
```

Comment:
I think we need to `return "", err` here and a bunch of similar ones below...

Reply:
Done... I think